fe1-web/package.json

```diff
@@ -9,7 +9,7 @@
     "web": "expo start --web",
     "prebuild-web": "node node_modules/expo-optimize/build/index.js",
     "build-web": "expo build:web",
-    "postbuild-web": "zip-build web-build -t %NAME%_%TIMESTAMP%.%EXT%",
+    "postbuild-web": "zip-build web-build -t fe1.zip && node build/web-postbuild.js",
```

Comment:
Did you erase the timestamp part? I think it was quite useful to be able to keep past versions of the build (and to locate them in time).

Reply:
That is true! Yes I needed to do this here so that the script knows the name of the file which should be renamed but I will add it back in `web-postbuild.js`.